controller/k8s/api.go

```diff
@@ -330,12 +330,24 @@ func (api *API) GetObjects(namespace, restype, name string) ([]runtime.Object, e
 // singular resource type (e.g. deployment, daemonset, job, etc.)
 func (api *API) GetOwnerKindAndName(pod *corev1.Pod) (string, string) {
 	if len(pod.GetOwnerReferences()) != 1 {
```

Comment:
TIOLI: there are 4 calls to `pod.GetOwnerReferences()` here, how about setting that in some `ownerRefs` var?